services/blockstorage/sync/harness.go

```diff
@@ -12,49 +12,12 @@ import (
 	"github.com/orbs-network/orbs-spec/types/go/protocol/gossipmessages"
 	"github.com/orbs-network/orbs-spec/types/go/services"
 	"github.com/orbs-network/orbs-spec/types/go/services/gossiptopics"
+	"github.com/pkg/errors"
 	"github.com/stretchr/testify/require"
 	"testing"
 	"time"
 )
 
-// the storage mock should be moved to its own file, but i have a weird goland bug where it will not identify it and its driving me mad, putting this here for now
```

Comment:
I assume the reason it didn't work last time was that the new file was named `storage_mock_test.go` and not `storage_mock.go`

Unfortunately we can't name it `_test`